### ISSUE TITLE
Incorrect case for character literal with kind

### ISSUE THREAD

**Issue by tskeith** [flang-compiler/f18#348]
When a character literal has a kind parameter, the prescanner appears to incorrectly convert the characters to lower case. In this example, `1_"ABC"` is changed to `1_"abc"`.

```
$ cat t.f90
character(3), parameter :: a = "ABC";
character(3), parameter :: b = 1_"ABC";
end

$ f18 -E t.f90
character(3), parameter :: a = "ABC";
character(3), parameter :: b = 1_"abc";
end
```

**Comment by klausler** [comment]
Believed to be fixed by PR#349, please close if you concur; thanks!

**Comment by tskeith** [comment]
Fixed, thank you.
